lib/core/include/irods/rcMisc.h

```diff
@@ -349,20 +339,16 @@ int getNumFilesInDir(const char* mydir);
 
 int getRandomArray(int** randomArray, int size);
 
-// Issue 3988: replaces isPathSymlink() function below which is soon to be deprecated.
 // Returns:
 //         0 - treat the parameter path as NOT a symlink
 //         1 - treat the parameter path as a symlink
 //        <0 - Error code (message in the message stack)
 int isPathSymlink_err(rodsArguments_t* rodsArgs, const char* path);
 
-// Issue 3988: will be DEPRECATED in a future release in favor of the
-// function above (isPathSymlink_err). The isPathSymlink() function does not return error
-// codes, although it will print error messages as needed.
-// Returns:
-//         0 - treat the parameter path as NOT a symlink
-//         1 - treat the parameter path as a symlink
-int isPathSymlink(rodsArguments_t* rodsArgs, const char* path); // DEPRECATED in a future release
+// clang-format off
+__attribute__((deprecated))
+int isPathSymlink(rodsArguments_t* rodsArgs, const char* path);
```

Comment:
Open an issue to remove this function in iRODS 6 and link it to the deprecation issue you opened.

Reply:
Opened #8718